src/components/explosions/TankExplosion.module.scss

```diff
@@ -1,16 +1,17 @@
 .TankExplosion {
-  width: var(--double-block-size);
-  height: var(--double-block-size);
-  background: center / contain;
-  animation: tankExplosion 2s;
+  --tank-explosion-bg-x: calc(0px - var(--bg-double-block-size) * 19);
+  --tank-explosion-bg-y: calc(0px - var(--bg-double-block-size) * 8);
+
+  width: calc(2*var(--double-block-size));
+  height: calc(2*var(--double-block-size));
```

Comment:
As much as I know, calc requires that you have space around operators, which can be a wrong syntax. Did you test this? If not, test all your PRs first.

- [ ] test PR and most probably fix this.